Code/Tools/AssetProcessor/native/AssetManager/PathDependencyManager.cpp

```diff
@@ -514,6 +514,7 @@ namespace AssetProcessor
 #if defined(CARBONATED)
             bool isRecursiveDependency = dependencyPathSearch.ends_with(RecursiveDependenciesPattern);
             dependencyPathSearch = isRecursiveDependency ? dependencyPathSearch.substr(0, dependencyPathSearch.length() - 1) : dependencyPathSearch;
+            AZStd::string pathWildcardSearchPathWithPlatform(platform + AZ_CORRECT_DATABASE_SEPARATOR_STRING + dependencyPathSearch);
 #endif
```

Comment:
Why do we need "platform" there?  Compared to "dependencyPathSearch ", does "dependencyPathSearch" have the platform added into it?

Reply:
OK, so what's the reason you're adding a wildcard search _with_ platform here?  Can you please explain the logic/process here?

Reply:
Why is the platform not prepended elsewhere?  Even in the original behaviour, with the /* pattern, the platform was not prepended.  Is this a bug in the engine dependency resolution or ours?

Reply:
got it, so we don't need to prepend with the platform?  Will it work without the platform specific wildcard search?

Reply:
I see - so why not just use PathView?